modular_zubbers/code/modules/mob/living/carbon/human/species_types/felinid.dm

```diff
@@ -0,0 +1,10 @@
+/datum/species/human/felinid
+	use_skintones = FALSE
```

Comment:
```suggestion
	use_skintones = NO_SKINTONES
```

Reply:
This is not actually a boolean

Reply:
Doing this gives me an undefined Var on compile, and looking at _species thats how its default defined.